docs/content/concepts/partitions-schedules-sensors/schedules.mdx

```diff
@@ -78,23 +78,31 @@ If you don't need access to the context parameter, you can omit it from the deco
+The Partitioned Jobs concepts page includes an [example of how to define a date-partitioned job](/concepts/partitions-schedules-sensors/partitions#date-partitioned-job). Having defined that job, you can construct a schedule for it using <PyObject object="build_schedule_from_partitioned_job"/>. For example:
 
 ```python file=/concepts/partitions_schedules_sensors/schedule_from_partitions.py startafter=start_marker endbefore=end_marker
-from dagster import build_schedule_from_partitioned_job
+from dagster import build_schedule_from_partitioned_job, job
 
-do_stuff_partitioned_schedule = build_schedule_from_partitioned_job(do_stuff_partitioned)
+
+@job
+def do_stuff_partitioned():
+    ...
+
+
+do_stuff_partitioned_schedule = build_schedule_from_partitioned_job(
+    do_stuff_partitioned,
+)
 ```
 
-Note that when the schedule submits a run on a particular day, it will typically be for the partition whose key corresponds to previous day. For example, the schedule would kick off the `2020-04-01` partition on `2020-04-02`. That's because each partition corresponds to a time window. The key of the partition is the start of the time window, but the schedule waits until the end of the time window to kick off the run. Kicking off a run at the end of a time window means the run can process data from within that time window.
+Each schedule tick of a partitioned job fills in the most recent partition in the list of partitions. Note that this implies that when the schedule submits a run on a particular day, it will typically be for the partition whose key corresponds to the previous day. For example, the schedule will fill in the `2020-04-01` partition on `2020-04-02`. That's because each partition corresponds to a time window. The key of the partition is the start of the time window, but the partition isn't included in the list until its time window has completed. Kicking off a run at the end of the time window means the run can process data from within that entire time window.
 
-Similar is true for hourly-, weekly-, and monthly- partitioned jobs.
+However, you can use the `end_offset` parameter of <PyObject object="daily_partitioned_config" decorator /> to change which partition is the most recent partition that is filled in at each schedule tick. Setting `end_offset` to `1` will shift the partitions forward so that the schedule tick that runs on day `N` will fill in day `N`'s partition instead of day `N-1`, and setting `end_offset` to a negative number will cause the schedule to fill in earlier days' partitions. In general, setting `end_offset` to `X` will cause the partition that runs on day `N` to fill in the partition for day `N - 1 + X`. The same holds true for hourly, weekly, and monthly partitioned jobs, for their respective partition sizes.
```

Comment:
I think it would be helpful to document the declarative rule that governs what's going on here: something like "Selects the latest partition in the partition set that exists as-of the scheduled execution time."

Also, consider replacing "shift" with "extend" or similar, because the parameter adds partitions at the end, it doesn't remove partitions at the beginning.

Also, I think this conflicts with the docstring on `latest_window_partition_selector`, though not the implementation. It's worth updating that docstring.

Reply:
incorporated!